arch/riscv/Kconfig

```diff
@@ -391,6 +391,9 @@ config GEN_IRQ_VECTOR_TABLE
 config ARCH_HAS_SINGLE_THREAD_SUPPORT
 	default y if !SMP
 
+config ARCH_HAS_DIRECTED_IPIS
+	default y
```

Comment:
Nitpick: this wants to be a select, not a default.  It's not a tunable an app is expected to change, the architecture always provides the API

Reply:
That was a leftover artefact from previous development revisions. Removed.